ci-operator/step-registry/ipi/conf/ibmcloud/custom-endpoints/ipi-conf-ibmcloud-custom-endpoints-ref.yaml

```diff
@@ -10,6 +10,9 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  dependencies:
+  - name: "release:latest"
+    env: RELEASE_IMAGE_LATEST_FROM_BUILD_FARM
```

Comment:
We do not need this env.

Reply:
based on the test https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/58984/rehearse-58984-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.17-installer-rehearse-ibmcloud-dis/1861637401475026944/artifacts/installer-rehearse-ibmcloud-dis/ipi-conf-ibmcloud-custom-endpoints/build-log.txt 
I need to restore checking the isPre418, then RELEASE_IMAGE_LATEST_FROM_BUILD_FARM will be used in isPreVersion function

Reply:
Why ?

I explained in https://github.com/openshift/release/pull/58984#discussion_r1859780081 and https://github.com/openshift/release/pull/58984#discussion_r1859785376. 

`ipi-conf-ibmcloud-custom-endpoints` is just an unit step, it is only used to overwrite endpoints  in install-config.ymal, it does not care about if the OCP version support the endpoints or not. If users only want to patch the new endpoints introduced in 4.18, users need to add the new endpoints setting in job config level or create a new install workflow.

Reply:
[cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private-provision](https://steps.ci.openshift.org/chain/cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private-provision) use   _ipi-conf-ibmcloud-custom-endpoints_ to add the endpoints. 
Need to add the new workflow for test eps?

Reply:
I will try to move the new eps to ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.18.yaml and ./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml

Reply:
We can easily run different ep testing in different versions of OCP by using ENV setting, e.g:
4.18-nightly.yaml:
```
- as: ibmcloud-ipi-disc-priv-f28
  steps:
    cluster_profile: ibmcloud-qe-2
    env:
      SERVICE_ENDPOINT_COSConfig: "DEFAULT_ENDPOINT"
    test: 
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private
```
4.14-nightly.yaml:
```
- as: ibmcloud-ipi-disc-priv-f28
  steps:
    cluster_profile: ibmcloud-qe-2
    test: 
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private
``